osu.Game.Rulesets.Osu/Mods/OsuModAlternate.cs

```diff
@@ -89,9 +102,22 @@ private bool checkCorrectAction(OsuAction action)
             {
                 // User alternated correctly.
                 lastActionPressed = action;
+                numActions = 0;
                 return true;
             }
 
+            if (lastActionPressed == action)
+            {
+                // If same action, increment counter.
+                numActions++;
```

Comment:
nit: would probably rename to something like `lastActionPressCount` or something